PTC: User Documentation (#20)
bssw-psip/ptc-catalog: catalog/UserDocumentation.md

# End-user Documentation

## Target

> Creating end-user documentation for a software project

Reviewer: ```suggestion
Create end-user documentation for a software project
```

## User Story

> As application developers, we want to document our project, so that new users will try the application.

Author: ```suggestion
As an application developer, I want to document our project, so that new users will try the application.
```

> As application developers, we want to document our project, so that new users will try the application.

Reviewer: Do we want to use "we" here or enforce the the "I" terminology? I'm happy with we... was our decision in #5 that either is okay?

Author: Yes, it should be "I".

## Card

| Score         | Description |
| :-------------: | :------------- |
| 0 | Little or no project documentation exists. |
| 1 | An initial set of documentation has been created<sup>1,2</sup>. |
| 2 | Feedback from users is collected on a regular basis and is used to improve documentation<sup>3</sup>.  |
| 3 | Scripts and/or auto-documentation tools are used to ensure the documentation is kept up to date. |
| 4 | Documentation maintenance is incorporated into the project's continuous integration process. |

## Comments

1. A good method for staring is to investigate what documentation is provided by other contemporary similar projects. 
Inspired by research results, create a scope of your eventual ideal end-user documentation.
2. Focus on creation of initial set of documentation that is most likely to be used by end-users (ex: README, software 
introduction guide, software installation and initial setup testing guide, troubleshooting guide etc.

> 3. Develop a process to solicit feedback on the user documentation and incorporate improvements.

Author: ```suggestion
3. Develop a process to solicit feedback on the user documentation and incorporate improvements.
4. There are many popular technologies for hosting documentation. These include Read The Docs, GitHub/GitLab Pages, GitBook, docsify, or simply creating a user manual using LaTex.
```

> 3. Develop a process to solicit feedback on the user documentation and incorporate improvements.

Reviewer: I think we need one more comment which lists some common technologies, at least readthedocs, user manual created with latex, maaaaybe doxygen.

Author: Please add.
